src/controllers/encodingsettings.js

```diff
@@ -14,6 +14,7 @@ define(["jQuery", "loading", "globalize", "dom", "libraryMenu"], function ($, lo
         $("#txtVaapiDevice", page).val(config.VaapiDevice || "");
         page.querySelector("#selectEncoderPreset").value = config.EncoderPreset || "";
         page.querySelector("#txtH264Crf").value = config.H264Crf || "";
+        page.querySelector("#selectDeinterlaceMethod").value = config.DeinterlaceMethod || "";
```

Comment:
If `config.DeinterlaceMethod` is not set, `selectDeinterlaceMethod` is empty. Should there be a default method?

Reply:
It should always be set, the default server-side value is "yadif" per this code: https://github.com/jellyfin/jellyfin/commit/5acd0521a26cbc89c66cc20cad3213b4d2a111ee#diff-d0f215ed6b3deb7472d977cd0fe0ff7e

We could add a default in encodingsettings, but there isn't one for most of the other settings (EncoderPreset, H264Crf, etc) which are already there.